platform-controller/src/main/java/org/hobbit/controller/docker/ContainerManagerImpl.java

```diff
@@ -452,6 +452,9 @@ private String createContainer(String imageName, String containerType, String pa
             LOGGER.warn("The swarm cluster got only 1 node, I will not use placement constraints.");
         }
 
+        // add hardware information to environment
+        defaultEnv.add(Constants.HARDWARE_NUMBER_OF_NODES_KEY + "=" + numberOfSwarmNodes);
```

Comment:
The given information is a little bit too general. For example, in our online cluster, a benchmarked system would get the information that 8 hardware nodes are available. However, only 3 of them are available for the system. We should have a better solution for that, i.e., one variable for the number of nodes avialable for the system and a variable for the benchmark.

Reply:
Added separate variables for system and benchmark nodes. To prevent further cluttering there, let's add other information to RDF model which would be added next.